search-books/src/main/java/nix/education/util/FacetFields.java

```diff
@@ -0,0 +1,10 @@
+package nix.education.util;
+
+public class FacetFields {
```

Comment:
This can be an interface or a final class. Rather an interface. Also I'd name it FacetsConstants.

Reply:
Rename to SearchConstants and changed to interface from class.